plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/utils.ts

```diff
@@ -31,3 +32,12 @@ export const hasChildren = ( children ): boolean => {
 		return isObject( child ) && child.key !== null;
 	} );
 };
+
+/**
+ * Gets the total items description text from PHP-computed setting.
+ *
+ * @return {string} The description text for the total items.
```

Comment:
I think this is still saying "total items" and not "totals item", which we might want to update for consistency.


```suggestion
 * Gets the totals item description text from PHP-computed setting.
 *
 * @return {string} The description text for the totals item.
```